src/main/java/com/iridium/iridiumskyblock/bank/CrystalsBankItem.java

```diff
@@ -51,13 +51,13 @@ public double withdraw(Player player, Number amount) {
                         player.getWorld().dropItem(player.getLocation(), itemStack)
                 );
                 player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().bankWithdrew
-                        .replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix))
+                                .replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix))
```

Comment:
Whats with the random indentations?

Reply:
it makes better code Hey Chris you can't understand it stoopid